product-service/__tests__/handlers/getProductById.test.ts

```diff
@@ -1,46 +0,0 @@
-import axios from 'axios';
```

Comment:
Why did you delete this test suite?

Reply:
It was implemented for local data files and is not working with PostgreSQL